packages/labs/analyzer/src/lib/javascript/variables.ts

```diff
@@ -49,8 +49,9 @@ export const getVariableDeclarationInfo = (
   statement: ts.VariableStatement,
   analyzer: AnalyzerInterface
 ): DeclarationInfo[] => {
+  const isExport = hasExportKeyword(statement);
```

Comment:
We were previously checking for the `export` keyword on the wrong node for variable declarations; it will be on the top-level `VariableDeclarationList` statement, rather than on `VariableDeclaration` (the latter are the individual names in a potential destructuring declaration ala `export const {foo, bar} = obj`.